submarine-sdk/pysubmarine/submarine/job/README.md

```diff
@@ -0,0 +1,31 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# How to generate Experiment API
```

Comment:
How about moving this document to "submarine-sdk/pysubmarine"? Because this will be used to generate all  APIs and not only for "experiment". It will have for instance, the "environment" API later.

Reply:
Agree. updated